app/build.gradle

```diff
@@ -19,11 +20,12 @@ ext.getLocalProperty = { propertyName ->
 
 android {
     compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    buildToolsVersion "26.0.3"
 
     defaultConfig {
         applicationId "io.intrepid.skotlinton"
         minSdkVersion 17
+        //noinspection OldTargetApi
```

Comment:
Why not just upgrade everything to 27?

Reply:
If I could cast a vote I would say
```
Compile 27
min 21
target 25
```

Reply:
We should always target the latest stable version since this will be used when starting new projects